src/js/post-911-gib-status/components/EnrollmentHistory.jsx

```diff
@@ -8,8 +8,7 @@ import { formatDateShort, formatDateLong } from '../utils/helpers';
 
 class EnrollmentHistory extends React.Component {
   render() {
-    const { enrollmentData } = this.props || {};
-    const today = formatDateShort(new Date());
+    const enrollmentData = this.props.enrollmentData || {};
```

Comment:
I don't think this was working the way it was before. `enrollmentData` was being set to `null` when `this.props.enrollmentData` was null, when it should have been set to `{}` in that case. This was creating problems where calls to properties within `enrollmentData` would fail (e.g., `enrollmentData.percentageBenefit`) and nothing would render. This change fixes that.